redisinsight/ui/src/components/markdown/CodeButtonBlock/components/RunConfirmationPopover.tsx

```diff
@@ -67,7 +68,7 @@ const RunConfirmationPopover = ({ onApply }: Props) => {
         aria-label="checkbox do not show agan"
       />
       <div className={styles.popoverFooter}>
-        <div>
+        <Row gap="m" justify="end">
```

Comment:
Bonus Points: Not related to the changes of the popover colors, but I fixed this minimal spacing issue as well.

| Before | After |
| - | - |
<img width="468" height="462" alt="Image" src="https://github.com/user-attachments/assets/a625a0ae-64bb-4139-94a1-51f939099a87" />|<img width="470" height="449" alt="Image" src="https://github.com/user-attachments/assets/6298995a-e1ab-482d-935d-0b0edb843947" />